app/services/data_exports/aggregate_metadata_file_generator.rb

```diff
@@ -0,0 +1,68 @@
+require 'csv'
+
+module DataExports
+  # Helper class for aggregating metadata from individual transcriptions
+  # within a group/workflow/project into a single csv file
+  class AggregateMetadataFileGenerator
+    def initialize(output_folder)
+      @metadata_rows = []
+      @metadata_file = File.join(output_folder, 'transcriptions_metadata.csv')
+    end
+
+    # Public: add metadata csv file to group folder
+    def generate_group_file(transcriptions)
```

Comment:
Not blocking: These public methods seem to be distinct use cases for generating transcription data exports. I could call these from the same instance and possibly mix the data in the resulting csv file via `@metadata_rows` and `@metadata_file` instance vars. 

It might be better to split these into different classes or convert them to class (static) methods that don't share instance state to avoid this happening.

Reply:
Oh I didnt think of this!

Reply:
Ended up changing it so that all the methods are class methods.